presto-main/src/main/java/io/prestosql/server/ClusterStatsResource.java

```diff
@@ -57,9 +58,10 @@ public ClusterStats getClusterStats()
         long blockedQueries = 0;
         long queuedQueries = 0;
 
-        long activeNodes = nodeManager.getNodes(NodeState.ACTIVE).size();
-        if (!isIncludeCoordinator) {
-            activeNodes -= 1;
+        long activeNodes = Sets.difference(nodeManager.getNodes(NodeState.ACTIVE),
```

Comment:
The existing code it just wrong since having multiple coordinators is allowed today.  This section is supposed to being used to populate `ClusterStats.activeWorkers`, but we are assuming there is always exactly one active coordinator.  I think we should change this code to:
```java
        long activeWorkers = nodeManager.getNodes(NodeState.ACTIVE).stream()
                .filter(node -> isIncludeCoordinator || !node.isCoordinator())
                .count();
```

Then as a separate commit (or PR), I think we should add `ClusterStats.activeCoordinators`, so you can just "know" what the current cluster setup without having to infer it from the stats.  BTW when adding the `activeCoordinators` field, please place it immediately before the existing `activeWorkers` field in `ClusterStats`.

Reply:
@dain ，thanks  very much ，I will submit the code later . I am a fresh to Presto, thanks  for your guidance  again!